python/hsfs/engine/hive.py

```diff
@@ -249,45 +253,19 @@ def save_dataframe(
 
         # Launch job
         print("Launching ingestion job...")
-        self._job_api.launch(ingestion_job.job.name)
+        fg_job = self._job_api.launch(ingestion_job.job.name)
```

Comment:
this is where I think it breaks cause `job_api.launch()` doesn't return the job.

Reply:
True, I noticed that I already have the job object in `ingestion_job.job` so I'm using that one instead and reverting the changes on this line.

Also as mentioned in the other comment, `launch` returns an execution not a job